invenio_app_rdm/users_ui/templates/semantic-ui/invenio_app_rdm/users/header.html

```diff
@@ -12,7 +12,7 @@
     <div class="ui items unstackable">
       <div class="item">
         <div class="ui image dashboard-header-avatar">
-          <img src="{{ user_avatar }}"/>
+          <img src="{{ user_avatar }}" alt="" />
```

Comment:
Is it bad practice to have here `alt="{{ _("User's avatar") }}"` or some such?

Reply:
Good question... I'd consider this decorative rather than informative since it adds no functionality or meaning but this could be debated.  https://www.w3.org/WAI/tutorials/images/decorative/#example-3-image-with-adjacent-text-alternative